src/sources/internal_logs.rs

```diff
@@ -100,8 +115,20 @@ async fn run(
             log.insert(host_key.as_str(), hostname.to_owned());
         }
         log.insert(pid_key.as_str(), pid);
-        log.try_insert(log_schema().source_type_key(), Bytes::from("internal_logs"));
-        log.try_insert(log_schema().timestamp_key(), Utc::now());
+
+        log_namespace.insert_vector_metadata(
+            &mut log,
+            log_schema().source_type_key(),
+            "source_type",
+            "internal_logs",
+        );
+        log_namespace.insert_vector_metadata(
+            &mut log,
+            log_schema().timestamp_key(),
+            "ingest_timestamp",
+            Utc::now(),
+        );
```

Comment:
`insert_standard_vector_source_metadata()` here instead, use my sweet sweet helper!

Reply:
addressed in latest changeset